### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by emrosenf**
- Cleaned up language and formatting
- Added information about known limitations
- Added details about element parameters
- Changed example code to use ES2015 imports

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,16 +146,38 @@ var styles = StyleSheet.create({
...
+
+For example, instead of 
+```
+<Route name="register" component={Register} title="Register"/>
+```
+
+you might write:
+
+```
+<Route name="register" component={connect(selectFnForRegister)(Register)} title="Register"/>
```
Comment (MITDelian) [inline README.md]: Is this really a common practice? I feel like it's more common to call connect in the component file, not inside the routing file.

**Comment by MITDelian** [comment]
Looks pretty good! I think we could likely improve the formatting of the `Feature` section but this is a great step in the right direction

File: README.md [hunk 2]
```diff
@@ -114,16 +146,38 @@ var styles = StyleSheet.create({
...
-    * If some your Routes have common attributes, you may define Schema element and just use 'schema' attribute for 'route'
-3. In any app screen:
-    * var {Actions} = require('react-native-router-flux');
-    * Actions.ACTION_NAME(PARAMS) will call appropriate action and params will be passed to the route
+## Redux/Flux support
+This component is not opinionated and does not depend on any implementation of Flux or Redux.
+
+All route actions can be hooked by adding handlers for `Actions.onPush`, `Actions.onReplace`, `Actions.onPop` in your store(s).
+
+If a handler returns false route action is ignored. For Redux, you will need to 'connect' your component to your store.
```
Comment (MITDelian) [inline README.md]: If a handler returns false *the* route action is ignored?

File: README.md (hunk 1 above)
Comment (emrosenf) [inline README.md]: I took that from #56. Will you write up an example of how you are doing it?

Comment (MITDelian) [inline README.md]: Yeah I was just reading through that discussion

Comment (MITDelian) [inline README.md]: The way I've been doing it, which I think is more common practice than using `connect` inline has been like this:

```
export default connect((state) => ({
  email: state.data.ui.login.email,
  password: state.data.ui.login.password,
  status: state.data.ui.login.status,
  error: state.data.ui.login.error
}))(Login);
```

Comment (MITDelian) [inline README.md]: and then when you import the `Login` component it is already `connect`ed

File: README.md (hunk 2 above)
Comment (emrosenf) [inline README.md]: Yes, look at index.js#L133

Comment (MITDelian) [inline README.md]: sorry that was more of a grammatical comment. I think the word `the` should be inserted in that sentence is all

Comment (MITDelian) [inline README.md]: `If a handler returns the false route action is ignored.`

**Comment by emrosenf** [comment]
Fixed the typo, thanks

**Pull request merged**